.github/workflows/publish_wheels.yml

```diff
@@ -0,0 +1,54 @@
+name: Build and Publish
+
+on:
+  workflow_dispatch:
+
+jobs:
+  deploy:
```

Comment:
publish would make more sense than deploy I guess

Reply:
I was planning to rename the job and forgot, good catch.